### DISCUSSION TITLE
Where are the files automatically backed up by drawio?

### DISCUSSION THREAD

**Discussion by Caeoon-Kaihong** [jgraph/drawio#4272] [General]
I've saved the changes I made in draw.io, also I had the automatic backup option enabled. However, after the computer accidentally restart, I couldn't find the changes that I used to save and also the .bpk file in the usual path.

**Comment by OLAWADARE** [comment]
I completed three different jobs, I'm trying locate them on draw.io .please how do I see my previous work?
